pkg/exporter/sumologicexporter/graphite_formatter_test.go

```diff
@@ -225,7 +226,7 @@ func TestGraphiteMetrics(t *testing.T) {
 			gf, err := newGraphiteFormatter(tt.formatter)
 			require.NoError(t, err)
 
-			result := gf.metric2String(tt.metric)
+			result := gf.metric2String(tt.metricFunc(false))
```

Comment:
shouldnt be the same like in carbon_formatter?

Reply:
What do you mean?

Reply:
I mean we are doing `metric, attributes := tt.metricFunc()` for exactly the same problem (versus `tt.metricFunc(false)`)

Reply:
changed